etc/dbus-serialbattery/installble.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+## DO NOT TOUCH THIS ##
+install_service() {
+    mkdir -p /service/dbus-blebattery-$1
```

Comment:
Not sure about that, but is /service the correct folder for this? If i remember correctly in recent versions of venusos this is overwritten on every reboot?

Reply:
Yes, this is correct. It's being recreated via /data/rc.local